components/brave_welcome_ui/brave_welcome.html

```diff
@@ -13,11 +13,14 @@
 
   body {
     background: black;
```

Comment:
Since we have nice background fade-in now, I'm wondering if it'll be better to make the initial solid color background light / dark, and even match the frame color like the NTP does. If you think that idea will work for this page, then we already have this setup as a simple serve-time variable replacement.

On the frontend side:
```css
background: $i18n{backgroundColor};
```

And in c++:
```c++
AddBackgroundColorToSource(source, web_ui->GetWebContents());
```

See `brave_new_tab.html` and `brave_new_tab_ui.cc` for where that's added.

Reply:
This is quite nice. Added.